assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -404,6 +404,23 @@ table.wc-block-cart-items {
 			}
 		}
 	}
+
+	// Submit button is sticky to bottom of screen on mobile.
+	.wc-block-cart__submit-container {
+		background: $white;
+		bottom: 0;
+		box-shadow: 0 -10px 20px 10px transparentize($core-grey-light-700, 0.5);
+		left: 0;
+		padding: $gap;
+		position: fixed;
+		width: 100%;
+		z-index: 1;
+	}
+	// Hide the Storefront sticky footer on cart & checkout page, so it doesn't clash with sticky submit.
+	// Note - this will only work on woo "Cart" page.
+	.woocommerce-page.woocommerce-cart.theme-storefront .storefront-handheld-footer-bar {
+		display: none;
+	}
```

Comment:
I think it's a great idea hiding this bar in the _Cart_ page, I would even hide it in _Checkout_ page as well, how does it sound?

My only concern is that I wonder if this code belongs to the Storefront theme instead of here. Stores using WooCommerce with other themes which are not Storefront don't need to load this CSS. I also think it would be easy to maintain it there than in this repo.

Reply:
Agreed with this. I think we should avoid theme specific styling in the plugin unless absolutely necessary.

Reply:
I'll revert this and log a follow up issue. It is essential that we fix this for Storefront, which we could potentially do in storefront directly or in a "theme support" stylesheet (similar to woo core approach).

<img width="459" alt="Screen Shot 2020-03-11 at 9 45 33 AM" src="https://user-images.githubusercontent.com/4167300/76357728-173a9b00-637d-11ea-8d3b-83acfd3ea8dc.png">

Logged here: #1930